source/features/deep-blame.tsx

```diff
@@ -44,13 +44,16 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number):
 	return false;
 });
 
-function getCommitHash(commit: HTMLElement): string {
-	return (commit.nextElementSibling as HTMLAnchorElement).href.split('/').pop()!;
+function getCommitHash(commit: HTMLLinkElement): string {
+	return commit.href.split('/').pop()!;
 }
 
 async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): Promise<void> {
 	const blameLink = event.delegateTarget;
-	const {lineNumber, commit: pullRequestCommit, prNumber} = blameLink.dataset;
+	const currentParentElement = blameLink.closest('.blame-hunk')!;
```

Comment:
You could rename this to `blameHunk`. "Hunk" is the git term for "group of lines"
```suggestion
	const blameHunk = blameLink.closest('.blame-hunk')!;
```